doc/meetings/build-wg-meeting-2017-08-22.md

```diff
@@ -0,0 +1,133 @@
+- Gibson: So giving someone access usually involves an issue or a request, so we
+  could make that request a PR.
+- All: Sounds good
+- Action: Gibson to PR changes to Readme.
+- João: I think there’s already one for github-bot, so that should be fine.
+
+### Demo KeyBox at next WG Meeting [#806](https://github.com/nodejs/build/issues/806)
+
+- Gibson: We should probably postpone
+- Rod: Sure, but worried that if it misses a few meetings it’ll get lost
+- Michael: We could have a separate meeting
+- Gibson: Maybe same time one of the other weeks
+- Rod: How about we wait till next time, and then if it’s missed next week we
+  can arrange a special meeting.
+- All: Sounds good.
+
+### tools: add download testing [#804](https://github.com/nodejs/build/pull/804)
+
+- Gibson: Mostly wanted to discuss where this stuff should go.
+- Rod: I really like this method (clone build repo, run script)
+- Gibson: That’s what we’re doing internally and it works really well.
+- Gibson: So maybe put in in `jenkins/` to prepare for other Jenkins scripts?
+- All: sounds good
+- Rod: What else might go in there.
+- Gibson: That ties neatly into the next topic
+
+### Migrate job configuration to pipeline files [#838](https://github.com/nodejs/build/issues/838)
+
+- Gibson: Would be good to get stuff out of the config.xml Jenkins files and
+  into Github
+- Michael: Could you quickly cover the benefits of pipelines
+- Gibson: Basically write your config as a Groovy file, rather than as options
+  in the GUI. It should also allow a lot more flexibility.
+- Michael: Would it allow us to condense our scripts more (fewer pipelines than
+  jobs, e.g. the “npm test a module” jobs)
+- Gibson: Yes, it should do that.
+- Rod: I really like the idea in principle, but in practice I’ve found pipelines
+  a pain, and not particularly flexible, they seem like something bolted on
+  afterwards. We don’t want to get stuck with a suboptimal solution.
+- All: Yep, let’s take it gradually.
+- Gibson: We should try with one of the smaller WG projects, maybe the n-api CI
+  job that Refael is working on.
+- All: Sounds good.
+
+### Jenkins update requiring Java 8 [#775](https://github.com/nodejs/build/issues/775)
+
+- Rod: So at the moment we’re stuck on this version of Jenkins until we can get
+  all our machines on Java 8. I’ve upgraded some but it’s hard going. This might
+  also impact our ability to switch to pipelines. Is this going to be a
+  permanent thing?
+- Gibson: George Adams has experience with this, he updated all the
+  [AdoptOpenJDK](https://github.com/AdoptOpenJDK/openjdk-infrastructure)
+  machines to Java 8, and they work with a similar set of machines. I know
+  George has been keen to contribute to our Build WG, so maybe some of the
+  Ansible updates made there could be ported over if applicable.
+- Rod: Sounds good. It’s possible pipelines are all plugin based, and don’t
+  really depend on having the latest Jenkins anway.
```

Comment:
Typo: anyway

Reply:
Done, but please just edit directly.

Reply:
> Done, but please just edit directly.

Not sure if I could have done that as I'm not a member of this WG.